packages/bento-web/src/profile/components/ProfileInstance.tsx

```diff
@@ -102,6 +115,39 @@ export const ProfileInstance: React.FC<ProfileInstanceProps> = ({
     useState<boolean>(false);
 
   const [selectedTab, setSelectedTab] = useState<ProfileTab>(PROFILE_TABS[0]);
+  const tabEventProps = useMemo(
+    () =>
+      !profile
+        ? null
+        : {
+            tab: selectedTab.toLowerCase(),
+            user_id: profile.user_id,
+            username: profile.username,
+            is_my_profile: isMyProfile,
+          },
+    [selectedTab, profile, isMyProfile],
+  );
+
+  const hasLoggedPageViewEvent = useRef<boolean>(false);
+  useEffect(() => {
+    if (!profile || hasLoggedPageViewEvent.current) {
+      return;
+    }
+    hasLoggedPageViewEvent.current = true;
+    Analytics.logEvent('view_profile', {
+      user_id: profile.user_id,
+      username: profile.username,
+      is_my_profile: isMyProfile,
+    });
+  }, [profile, isMyProfile]);
+
+  // TODO: check if this render twice in production too
```

Comment:
<img width="558" alt="image" src="https://user-images.githubusercontent.com/32605822/187026601-0985c26a-c772-4aaf-8441-e695107999e3.png">
Turns out this was because of React's strict mode